api/namex/utils/queue_util.py

```diff
@@ -0,0 +1,50 @@
+import json
+import time
+import uuid
+from datetime import datetime, timezone
+
+from config import Config
+from namex.services import queue
+
+
+def create_cloud_event_msg(msg_id, msg_type, source, time, identifier, json_data_body):  # pylint: disable=too-many-arguments # noqa E501
+    # industry standard arguments for this message
+    """Creates a payload for the cloud event messages."""
+    cloud_event_msg = {
+        'specversion': '1.x-wip',
```

Comment:
the specversion should be `1.0.1`

Reply:
Not used anymore. I'm using queue_commons function now.